idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.composegears.valkyrie.util.extension
+
+import com.intellij.openapi.vfs.VirtualFile
+import io.github.composegears.valkyrie.parser.svgxml.util.isSvgExtension
+import io.github.composegears.valkyrie.parser.svgxml.util.isXmlExtension
+
+inline val VirtualFile.isSvg: Boolean
+    get() = extension?.isSvgExtension ?: false
```

Comment:
```suggestion
    get() = extension?.isSvgExtension == true
```

Reply:
Just a small thing. Looks good.

Reply:
ok, will leave with false for now)
thanks for review 🙂